common/data_refinery_common/models/batches.py

```diff
@@ -113,8 +116,11 @@ def _get_downloaded_name(self) -> str:
         path = urllib.parse.urlparse(self.download_url).path
         return os.path.basename(path)
 
+    def get_base_name(self) -> str:
+        return self.name.replace(("." + self.raw_format), "")
+
     def get_processed_name(self) -> str:
-        file_base = self.name.split(".")[0]
+        file_base = self.get_base_name()
```

Comment:
Because the original implementation wasn't correct for `Files` with formats that were more complicated than `.*`, i.e. `.tar.gz`. `get_base_name()` properly handles those cases as well.